graylog2-server/src/main/java/org/graylog2/alerts/types/FieldContentValueAlertCondition.java

```diff
@@ -124,4 +126,9 @@ protected CheckResult runCheck() {
     public String getDescription() {
         return "field: " + field + ", value: " + value;
     }
+
+    @Override
+    public List<Message> getSearchHits() {
+        return searchHits;
```

Comment:
Agreed! We should make it a copy just to guarantee that other callbacks receive a list that has not been modified.